flang/lib/Lower/ConvertExpr.cpp

```diff
@@ -376,7 +376,9 @@ class ExprLowering {
 
   template <int KIND>
   mlir::Value genval(const Fortran::evaluate::Concat<KIND> &op) {
-    TODO();
+    // return genCharacterConcatenate(converter, genval(op.left()),
```

Comment:
This commented code generates the library call, correct?

Reply:
Yes, that was a left over from my other attempt with runtime call, I have deleted it completely for now.